### ISSUE TITLE
Handle SCSS errors

### ISSUE THREAD

**Issue by gregpriday** [siteorigin/theme-build#3]
When using `build:dev`, it's possible that a SCSS error might come up while you're writing code. The issue is that this halts the Gulp script.

```
[18:24:39] Starting 'sass'...

events.js:141
      throw er; // Unhandled 'error' event
      ^
Error: sass/woocommerce/_archives.scss
Error: style declaration must contain a value
        on line 71 of sass/woocommerce/_archives.scss
>> 					pointer-events:;
   -------------------^

    at options.error (/Users/gpriday/Dropbox/Sites/dev/wp-content/themes/siteorigin-north/build/node_modules/gulp-sass/node_modules/node-sass/lib/index.js:277:32)
```

Ideally it should handle this error and continue running in the background. Maybe just displaying the error in the command line.

**Comment by adiraomj** [comment]
After displaying an error in the command line, `build:dev` seems to continue running. But it does not compile SCSS to css. I have to stop and run the command every time.

**Comment by braamgenis** [comment]
@adiraomj I don't see this behaviour. Could you paste the snippet that causes the error and then doesn't compile?

**Comment by adiraomj** [comment]
Suppose I add `padding:` and save the file. The output in the terminal is:

```
[15:23:48] Starting 'sass'...
[15:23:48] { [Error: sass/site/primary/_posts-and-pages.scss
Error: Invalid CSS after "	padding:": expected expression (e.g. 1px, bold), was "}"
        on line 23 of sass/site/primary/_posts-and-pages.scss
>> 	padding:
   ---------^
]
  formatted: 'Error: Invalid CSS after "\tpadding:": expected expression (e.g. 1px, bold), was "}"\n        on line 23 of sass/site/primary/_posts-and-pages.scss\n>> \tpadding:\n   ---------^\n',
  message: 'sass/site/primary/_posts-and-pages.scss\nError: Invalid CSS after "\tpadding:": expected expression (e.g. 1px, bold), was "}"\n        on line 23 of sass/site/primary/_posts-and-pages.scss\n>> \tpadding:\n   ---------^\n',
  column: 10,
  line: 23,
  file: '/Applications/MAMP/htdocs/wpinked/wp-content/themes/wpinked-elysium/sass/site/primary/_posts-and-pages.scss',
  status: 1,
  messageFormatted: '\u001b[4msass/site/primary/_posts-and-pages.scss\u001b[24m\nError: Invalid CSS after "\tpadding:": expected expression (e.g. 1px, bold), was "}"\n        on line 23 of sass/site/primary/_posts-and-pages.scss\n>> \tpadding:\n   ---------^\n',
  messageOriginal: 'Invalid CSS after "\tpadding:": expected expression (e.g. 1px, bold), was "}"',
  relativePath: 'sass/site/primary/_posts-and-pages.scss',
  name: 'Error',
  stack: 'Error: sass/site/primary/_posts-and-pages.scss\nError: Invalid CSS after "\tpadding:": expected expression (e.g. 1px, bold), was "}"\n        on line 23 of sass/site/primary/_posts-and-pages.scss\n>> \tpadding:\n   ---------^\n\n    at options.error (/Applications/MAMP/htdocs/wpinked/wp-content/themes/wpinked-elysium/build/node_modules/gulp-sass/node_modules/node-sass/lib/index.js:277:32)',
  showStack: false,
  showProperties: true,
  plugin: 'gulp-sass' }
```
Then if I change it to `padding: 20px;`. I get the following in the terminal.
```
[15:23:58] Starting 'clean'...
[15:23:58] Finished 'clean' after 8.38 μs
[15:23:58] Starting 'external-sass'...
[15:23:58] Starting 'external-less'...
[15:23:58] Finished 'external-sass' after 53 ms
[15:23:58] Finished 'external-less' after 51 ms
```

Any changes after error isn't compiled. Hope this is helpful. Let me know :-)

**Comment by braamgenis** [comment]
@adiraomj Awesome, found the issue and [fixed](0c4d231b3dbe5ec9072eb2de9707d599d414f7ca) in develop branch. :+1:

**Comment by dr-dimitru** [comment]
Fix a link to the commit: 0c4d231b3dbe5ec9072eb2de9707d599d414f7ca
